ethcore/src/block.rs

```diff
@@ -405,15 +404,20 @@ impl LockedBlock {
 	/// Provide a valid seal in order to turn this into a `SealedBlock`.
 	///
 	/// NOTE: This does not check the validity of `seal` with the engine.
-	pub fn seal(self, engine: &EthEngine, seal: Vec<Bytes>) -> Result<SealedBlock, BlockError> {
-		let expected_seal_fields = engine.seal_fields(&self.block.header);
```

Comment:
In case this confuses anyone -- `LockedBlock` implements a `Deref` to `ExecutedBlock` so both with `block.` and without are valid.